tutor/test_t1_18_CreateAnExternalAssignment.py

```diff
@@ -440,6 +436,8 @@ def test_teacher_save_a_draft_external_assignment_8089(self):
         today = datetime.date.today()
         opens_on = (today + datetime.timedelta(days=0)).strftime('%m/%d/%Y')
         closes_on = (today + datetime.timedelta(days=6)).strftime('%m/%d/%Y')
+        print(opens_on)
+        print(closes_on)
```

Comment:
insert some text before the variables so that an error report offers information about the text following it; otherwise it will just have two random dates to track down